### ISSUE TITLE
Influx WR Sounds not working properly

### ISSUE THREAD

**Issue by Dunkstormen** [TotallyMehis/Influx-Timer#61]
When I'm trying to configure the Influx Sounds module, I add the neccesary stuff to the influx_sounds.cfg file. But only the default voice from CSGO seems to work. Even though I've got the custom ones on both my client and on the server.

My sounds CFG:
`"Sounds"
{
	"bot\good_one.wav"
	{
		"other"		"1"
	}
	"sound\bot\nice_one_sir.wav"
	{
		"pb"		"1"
	}
	"sound\bot\owned.wav"
	{
		"best"		"1"
	}
}
`

I don't know if I'm doing something completely wrong or if there is an error in the module, since I get no errors when starting the server or when the sound is called.

**Comment by TotallyMehis** [comment]
Your sound config is the same as the default one? You need to show the real config.

**Comment by Dunkstormen** [comment]
https://pastebin.com/4HHSpjCV

The only thing I've changed is that I've remove the player\vo\phoenix\onarollbrag15.wav sound.

**Comment by TotallyMehis** [comment]
You have those sounds ripped from CSS and they are located on the server, possibly in a fast-dl and the clients download them fine?

**Comment by Dunkstormen** [comment]
Using some custom ones I had laying around, and I've made sure that the sounds are probably downloaded though my FastDL.

When playing them through a program like VLC they play fine, but when using the playvol command, the game recognizes and finds the sound file, but no sound is being emitted.

However some of the other sounds I've got can't be located using the playvol command for some odd reason.

EDIT:
Sometimes the playvol works and are able to locate the file (Without sound being emitted), and sometimes it's unable to locate the sound file. Tested with the default CSGO sounds, and works fine.
